src/log/test.cc

```diff
@@ -364,6 +365,7 @@ TEST(Log, GarbleRecovery)
   struct stat file_status;
   ASSERT_EQ(lstat(test_file, &file_status), 0);
   ASSERT_GT(file_status.st_size, 2000);
+  unlink(test_file);	// make sure that test_file is deleted
```

Comment:
remove this line. and replace the previous `unlink(test_file)` call with
```c++
auto remove_log = make_scope_guard([test_name] {
  unlink(test_name);
});
```

Reply:
@tchaikov 
I saw that construction in blkdev, and ws going to look it up, to see if it does what I think it does.